tests/llm/fixtures/shared/inject_datadog_alert_logs.py

```
@@ -0,0 +1,261 @@
+    )
+    with urllib.request.urlopen(req, timeout=30) as resp:
+        return len(json.loads(resp.read().decode("utf-8")).get("data", []))
+
+
+def wait_for_indexing(api_key: str, app_key: str, namespace: str,
+                      from_ms: int, to_ms: int, min_count: int) -> None:
+    for attempt in range(1, 13):
+        time.sleep(5 if attempt == 1 else 10)
+        try:
+            count = query_log_count(api_key, app_key, namespace, from_ms, to_ms)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)